src/hivpy/sex_behaviour_data.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import scipy.stats as stat
+
+# Baseline sexual risk for age and gender
+# age groups 15 <= age < 20
+#            20 <= age < 25
+#            ...
+#            60 <= age
+baseline_risk = np.array([[0.3,   1.8],
+                          [0.4,   1.8],
+                          [0.85,  1.0],
+                          [1.0,   0.8],
+                          [0.85,  0.5],
+                          [0.5,   0.35],
+                          [0.4,   0.3],
+                          [0.35,  0.1],
+                          [0.2,   0.03],
+                          [0.15,  0.02]])
+
+# Sexual behaviour transition matrices
+
+# Male
+# Probability of transitions between male sexual behaviour groups
+# 0 = Zero, 1 = Low, 2 = Medium, 3 = High
+sex_behaviour_trans_male_options = np.array([
+    [[0.995, 0.005, 0.005, 0.00005],
```

Comment:
Huh, I thought these had to sum to 1 but it seems they are normalised afterwards in the original model (I think for ease of expression and modification)

Reply:
See comment on the probabilities calculation below... not sure which is correct yet!